srcs/app/CGI/CgiInfo.cpp

```diff
@@ -0,0 +1,12 @@
+#include <CgiInfo.hpp>
+
+CgiInfo::CgiInfo() : pid(-1), cgi_output_fd(-1) {}
+
+CgiInfo::CgiInfo(const pid_t &new_pid,
```

Comment:
Why do you believe it's useful to pass these non aggregate values (new_pid and new_cgi_output_fd) by reference?

Reply:
I've changed it. I was used to passing the parameters by reference in the constructors.